components/mpas-framework/README.md

```diff
@@ -0,0 +1,60 @@
+MPAS-v7.0
```

Comment:
This file should be rewritten to describe the code that's actually in E3SM and how it's organized.  That could happen in an immediate follow-on PR.